Competitive_Programming/C++/Array/longest_sub_without_repeat.md

```diff
@@ -0,0 +1,60 @@
+# Length of the Longest Substring without Repeating Characters
+
+Here we will see a program to find the length of Longest substring without repeating characters.  
+
+## Explanation
+In this program we are using the concept of Window sliding i.e. Whenever we see repetition, we remove the previous occurrence and slide the window.  
+A loop is runniig from i=0 to n=sizeof(str).  
+In side this loop one more loop is running from j=i till n.  
+We define vector with all the value as false and keep it makin true for the characters that are visited once in the loop.
```

Comment:
Please explain the approach in detail with snippet examples as this is a basic problem.